x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -480,10 +484,27 @@ export function App({
         // Temporarily required until the 'by value' paradigm is default.
         (dashboardFeatureFlag.allowByValueEmbeddables || Boolean(initialInput))
     ),
+    enableExportToCSV: Boolean(
+      lastKnownDoc?.state?.activeData && Object.keys(lastKnownDoc.state.activeData).length
```

Comment:
Should we allow csv download if the current config is invalid? Not sure whether it would be better to disable the export button as long as `state.isSaveable` is false. Wdyt?

Reply:
Wasn't aware of that flag. Will update it.